content/en/docs/concepts/overview/working-with-objects/object-management.md

```diff
@@ -113,7 +113,7 @@ Disadvantages compared to imperative commands:
 Advantages compared to declarative object configuration:
 
 - Imperative object configuration behavior is simpler and easier to understand.
-- As of Kubernetes version 1.5, imperative object configuration is more mature.
+- As of Kubernetes version 1.5, imperative object configuration was highlighted for its maturity. However, in the present Kubernetes landscape, both imperative (kubectl create) and declarative (kubectl apply) object configurations are well-established and widely used in production environments.
```

Comment:
This is a section highlighting the advantages of the imperative object configuration. Therefore, IMO, even though the rephrasing is accurate, it doesn't fit this particular section.

Reply:
Okay should the point be related to this "As of Kubernetes version 1.5, imperative object configuration is more mature." or can I add any other advantage

Reply:
```suggestion
```

I'd drop the line.

Reply:
Okay

Reply:
```suggestion
```

Reply:
Feel free to make that change.